Can't run/build/test on Windows

<!--- Provide a general summary of the issue in the Title above -->

## Expected Behavior
I'm expecting the scripts defined in package.json to also work on Windows

## Current Behavior
Running e.g. `npm run demos:serve` I get the following output

```
$ npm run demos:serve

> britecharts@1.5.3 demos:serve C:\Users\tlb\dev\britecharts
> WEBPACK_ENV=demos webpack-dev-server --inline --hot --port 8001

'WEBPACK_ENV' is not recognized as an internal or external command,
operable program or batch file.

npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\tobbe\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "run" "demos:serve"
npm ERR! node v6.3.1
npm ERR! npm  v3.10.5
npm ERR! code ELIFECYCLE
npm ERR! britecharts@1.5.3 demos:serve: `WEBPACK_ENV=demos webpack-dev-server --inline --hot --port 8001`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the britecharts@1.5.3 demos:serve script 'WEBPACK_ENV=demos webpack-dev-server --inline --hot --port 8001'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the britecharts package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     WEBPACK_ENV=demos webpack-dev-server --inline --hot --port 8001
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs britecharts
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls britecharts
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\tobbe\dev\britecharts\npm-debug.log
```

## Possible Solution
Don't rely on the unix way of handling environment variables

## Steps to Reproduce (for bugs)
On Windows:
1. Clone the britecharts repo
2. Run `npm install`
3. Run `npm run demos:serve`

## Context
I was following the [contributing.md guide](https://github.com/eventbrite/britecharts/blob/master/CONTRIBUTING.md) 

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* Version used: 1.5.3
* Browser Name and version: Chrome, latest
* Operating System and version (desktop or mobile): Windows 10
* Link to your project: https://github.com/Tobbe/britecharts

--- a/package.json
+++ b/package.json
@@ -18,20 +18,20 @@
     }
   ],
   "scripts": {
-    "demos:compile": "WEBPACK_ENV=demos webpack --progress --colors",
-    "demos:watch": "WEBPACK_ENV=demos webpack --watch --progress --colors",
-    "demos:serve": "WEBPACK_ENV=demos webpack-dev-server --inline --hot --port 8001",
+    "demos:compile": "webpack --progress --colors --mode=demos",
+    "demos:watch": "webpack --watch --progress --colors --mode=demos",
+    "demos:serve": "webpack-dev-server --inline --hot --port 8001 --mode=demos",
     "styles": "grunt lint && grunt sass && grunt cssmin",
-    "docs": "WEBPACK_ENV=demos webpack && npm run styles && grunt docs",
+    "docs": "webpack --mode=demos && npm run styles && grunt docs",
     "prod": "npm run _prodUMD && npm run _prod",
     "untrack": "grunt untrack",
     "release": "npm run prod && grunt release:patch",
     "release:minor": "npm run prod && grunt release:minor",
     "release:major": "npm run prod && grunt release:major",
-    "test": "WEBPACK_ENV=test karma start",
-    "test:travis": "WEBPACK_ENV=test karma start --single-run --browsers PhantomJS",
-    "_prod": "WEBPACK_ENV=prod webpack",
-    "_prodUMD": "WEBPACK_ENV=prodUMD webpack",
+    "test": "karma start --mode=test",
+    "test:travis": "karma start --single-run --browsers PhantomJS --mode=test",
+    "_prod": "webpack --mode=prod",
+    "_prodUMD": "webpack --mode=prodUMD",
     "move": "grunt moveDemos",
     "watch": "grunt watch"
   },
@@ -110,6 +110,7 @@
     "webpack": "^1.13.3",
     "webpack-bundle-analyzer": "^2.3.1",
     "webpack-dev-server": "^1.16.2",
-    "webpack-livereload-plugin": "^0.9.0"
+    "webpack-livereload-plugin": "^0.9.0",
+    "yargs": "^8.0.1"
   }
 }

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -5,7 +5,7 @@ var webpack = require('webpack'),
     UglifyJsPlugin = webpack.optimize.UglifyJsPlugin,
     BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin,
 
-    env = process.env.WEBPACK_ENV, // dev | build
+    env = require('yargs').argv.mode,
     isProduction = env === 'prod' || env === 'prodUMD',
 
     chartModulesPath = path.resolve('./src/charts'),

